File: python/estructura_datos.py
estudiantes = [
    {"nombre": "Juan", "promedio": 4.5},
    {"nombre": "María", "promedio": 3.8},
    {"nombre": "Pedro", "promedio": 4.2},
    {"nombre": "Ana", "promedio": 2.0}
]
mejor = estudiantes[0]
for estudiante in estudiantes:
    if estudiante["promedio"] > mejor["promedio"]:
        mejor = estudiante
print('mejor estudiante', mejor.get("nombre"))
    

File: js/estructura_datos.js
let estudiantes = [
    {nombre: "Ana", promedio: 4.5},
    {nombre: "Luis", promedio: 3.8},
    {nombre: "Carlos", promedio: 4.9},
    {nombre: "María", promedio: 2.2}
];

let mejor = estudiantes[0];

for (let estudiante of estudiantes) {
    if (estudiante.promedio > mejor.promedio) {
        mejor = estudiante;
    }
}

console.log("Mejor estudiante:", mejor.nombre);